sim/core/buffs.go

```diff
@@ -765,7 +784,10 @@ func applyBuffEffects(agent Agent, playerFaction proto.Faction, raidBuffs *proto
 	if raidBuffs.BattleShout != proto.TristateEffect_TristateEffectMissing {
 		MakePermanent(BattleShoutAura(&character.Unit, GetTristateValueInt32(raidBuffs.BattleShout, 0, 5), 0))
 	}
-	if individualBuffs.BlessingOfMight != proto.TristateEffect_TristateEffectMissing && isAlliance {
+	
+	if raidBuffs.HornOfLordaeron && isAlliance {
+		character.AddStats(BuffSpellByLevel[HornOfLordaeron][level])
```

Comment:
Could you potentially create an aura for this? It just helps users feel confident that buffs are actually active. A lot of the old buff code still needs to be converted